README.md

```diff
@@ -23,4 +23,72 @@ This repository contains various specification related to the Ethereum 1.0 chain
 | Frontier Thawing | 200000 | 09/07/2015 | | | [Blog](https://blog.ethereum.org/2015/08/04/the-thawing-frontier/) |
 | Frontier | 1 | 07/30/2015 | | | [Blog](https://blog.ethereum.org/2015/07/22/frontier-is-coming-what-to-expect-and-how-to-prepare/) |
 
+## Consensus Specification (work-in-progress)
 
+ * [Rendered specification](https://quilt.github.io/eth1.0-specs/)
+
+## Usage
+
+The Ethereum specification is maintained as a Python library, for better integration with tooling and testing.
+
+Requires Python 3.7+
+
+### Building
+
+Building the documentation is most easily done through [`tox`](https://tox.readthedocs.io/en/latest/):
+
+```bash
+$ tox -e doc
+```
+
+The path to the generated HTML will be printed to the console.
+
+### Development
+
+Running the tests necessary to merge into the repository requires:
+
+ * Python 3.7.x (not 3.8 or later), and
+ * [PyPy 7.3.x](https://www.pypy.org/).
+
+These version ranges are necessary because, at the time of writing, PyPy is only compatible with Python 3.7.
+
+`eth1.0-specs` depends on a submodule that contains common tests that are run across all clients, so we need to clone the repo with the --recursive flag. Example:
+```bash
+$ git clone --recursive https://github.com/quilt/eth1.0-specs.git
+```
+
+Or, if you've already cloned the repository, you can fetch the submodules with:
+
+```bash
+$ git submodule update --init --recursive
+```
+
+The tests can be run with:
+```bash
+$ tox
+```
+
+The development tools can also be run outside of `tox`, and can automatically reformat the code:
+
+```bash
+$ pip install -e .[doc,lint,test]   # Installs eth1spec, and development tools.
+$ isort src                         # Organizes imports.
+$ black src                         # Formats code.
+$ flake8                            # Reports style/spelling/documentation errors.
```

Comment:
we have a convention in our python repositories of having a `MAKE` command that will do these things for you: [`make lint-roll`](https://github.com/ethereum/ddht/blob/142911d134ff839f3f79ff8fe9e45d3fe5a58cd0/Makefile#L29-L40)

Reply:
So far I've managed to avoid using `make` for anything ;)

`tox` does the checks, but as long as the actual build doesn't rely on `make`, I guess I'd be okay with a script to simplify auto-formatting.